docs/user_guide/backend_configuration.rst

```diff
@@ -0,0 +1,284 @@
+
+.. code-block:: python
+
+    from datetime import datetime
+    from uuid import uuid4
+
+    from neuroconv.tools.nwb_helpers import get_default_backend_configuration
+    from pynwb import NWBFile, TimeSeries
+
+    session_start_time = datetime(2020, 1, 1, 12, 30, 0)
+    nwbfile = NWBFile(
+        identifier=str(uuid4()),
+        session_start_time=session_start_time,
+        session_description="A session of my experiment.",
+    )
+
+    time_series = TimeSeries(
+        name="MyTimeSeries",
+        description="A time series from my experiment.",
+        unit="cm/s",
+        data=[1., 2., 3.],
+        timestamps=[0.0, 0.2, 0.4],
+    )
+    nwbfile.add_acquisition(time_series)
+
+    default_backend_configuration = get_default_backend_configuration(
+        nwbfile=nwbfile, backend="hdf5"
+    )
+
+From which a printout of the contents looks like...
+
+.. code-block:: python
+
+    print(default_backend_configuration)
+
+.. code-block:: bash
+
+    Configurable datasets identified using the hdf5 backend
+    -------------------------------------------------------
+
+    acquisition/MyTimeSeries/data
+    -----------------------------
+      dtype : float64
+      full shape of source array : (3,)
+      full size of source array : 24 B
+
+      buffer shape : (3,)
+      expected RAM usage : 24 B
+
+      chunk shape : (3,)
+      disk space usage per chunk : 24 B
+
+      compression method : gzip
+
+    acquisition/MyTimeSeries/timestamps
+    -----------------------------------
+      dtype : float64
+      full shape of source array : (3,)
+      full size of source array : 24 B
+
+      buffer shape : (3,)
+      expected RAM usage : 24 B
+
+      chunk shape : (3,)
+      disk space usage per chunk : 24 B
+
+      compression method : gzip
+
+Customization
+-------------
+
+To modify the chunking or buffering patterns and compression method or options, change those values in the ``.dataset_configurations`` object using the location of each dataset as a specifier.
+
+Let's demonstrate this by modifying everything we can for the ``data`` field of the ``TimeSeries`` object generated above...
+
+.. code-block:: python
+
+    dataset_configurations = default_backend_configuration.dataset_configurations
+    dataset_configuration = dataset_configurations["acquisition/MyTimeSeries/data"]
+    dataset_configuration.chunk_shape = (1,)
+    dataset_configuration.buffer_shape = (2,)
+    dataset_configuration.compression_method = "Zstd"
+    dataset_configuration.compression_options = dict(clevel=3)
+
+Some details to note about what can be changed...
+
+.. note::
+
+    Core fields such as the maximum shape and data type of the source data cannot be altered using this method.
+
+.. note::
+
+    The ``buffer_shape`` must be a multiple of the ``chunk_shape`` along each axis.
+
+.. note::
+
+    You can see what compression methods are available on your installation by examining the following...
```

Comment:
```suggestion
Core fields such as the maximum shape and data type of the source data cannot be altered using this method. The ``buffer_shape`` must be a multiple of the ``chunk_shape`` along each axis.

You can see what compression methods are available on your installation by examining the following:
```

Reply:
I like to reserve the "note" feature for side comments and gotchas. I feel like this is more essential information so I think it goes better in the main text. How would I have a dataset be stored in contiguous mode? Is that possible? Would I delete the entry from `default_backend_configuration.dataset_configurations`?

Reply:
Good point, I moved all of these and included your latest question into an FAQ section after the main 'how to' demonstrations